### PR TITLE
ADDED: ?edition=US param, needs new env var: frontPageIdUS

### PR DESCRIPTION
**Pull request by railsagainstignorance**
default is still UK homepage, but can override with ?edition=US or ?edition=INTL

needs an env var: frontPageIdUS=... for the CAPI id of the US homepage

### CODE REVIEW

File: server/lib/topStories.js [hunk 1]
```diff
@@ -4,11 +4,13 @@ const R = require('ramda');
 const briefArticleMapper = R.map(briefArticle);
 
 const siteApiEndpoint = process.env.siteApiEndpoint;
-const frontPageId = process.env.frontPageId;
-const mainContentKey = process.env.mainContentKey;
-const apiKey = process.env.apiKey;
+const mainContentKey  = process.env.mainContentKey;
+const apiKey          = process.env.apiKey;
+const frontPageIdUK   = process.env.frontPageId;
```
Comment (JakeChampion) [inline server/lib/topStories.js]: @railsagainstignorance In a prior PR you changed these strings from being env vars to being within the codebase itself as it was shown that they are not env specific. Could we apply that work into the PR? Here is the reference PR: https://github.com/ftlabs/big-ft/pull/11/files#diff-c9b6d92f13b7e31c43665de5383a27f0R20

File: server/lib/topStories.js [hunk 2]
```diff
@@ -1,29 +1,39 @@
...
 const briefArticle = require('./viewModel');
 const R = require('ramda');
 const briefArticleMapper = R.map(briefArticle);
 
 const siteApiEndpoint = process.env.siteApiEndpoint;
-const frontPageId = process.env.frontPageId;
 const mainContentKey = process.env.mainContentKey;
 const apiKey = process.env.apiKey;
+const frontPageIdUK = process.env.frontPageId;
+const frontPageIdUS = process.env.frontPageIdUS;
```
Comment (JakeChampion) [inline server/lib/topStories.js]: https://github.com/ftlabs/big-ft/pull/74#discussion_r53204591

**Comment by JakeChampion** [comment]
Other than my comment about switching from environment variables to adding the strings to the codebase itself, this looks good to me.

**Comment by AdaRoseCannon** [comment]
NOTE! siteApiEndpoint now apiOrigin as from that PR

File: server/lib/topStories.js (hunk 2 above)
Comment (railsagainstignorance) [inline server/lib/topStories.js]: yes, good point. the only id that needs to be secret is the apikey.
Can't see where in history the pageId was made explicit.
Will make the change.

**Pull request merged**